k2/csrc/host/rmepsilon.cc

```diff
@@ -22,6 +22,8 @@
 #include "k2/csrc/host/fsa.h"
 #include "k2/csrc/host/properties.h"
 #include "k2/csrc/host/util.h"
+#include "k2/csrc/macros.h"
```

Comment:
@csukuangfj Can you revert changes in `rmepsilon.cc`, I can add NVTX stuff in my next PR (I have a big change in this file).

Reply:
Fixed.

Reply:
Many thanks